proto/index.proto

```diff
@@ -29,8 +32,7 @@ message IndexRequest {
   // Ex. "gr12345" or "anon", or "public" or "foobarbaz".
   string namespace = 4;
 
-  // The strategy to use when indexing this repo.
-  // By default the server will assume REPLACE_REPO.
+  // Required. The strategy to use when indexing this repo.
```

Comment:
If you wanted to avoid the extra RPCs you could add a DELETE_NAMESPACE replacement strategy and handle this in IndexRequest. Slightly corny but /shrug

Reply:
Yeah, probably a bit cleaner overall to keep it consolidated to a single RPC... it's a bit janky to combine these, because the scope is different (namespace vs repo), but it's not totally clear what the best form factor for indexing RPCs will be in the future so I'll go with simpler / smaller footprint for now. I'll ping when it's ready again.